example/memcached-operator/controller/simple_resource_router.go

```diff
@@ -0,0 +1,49 @@
+package controller
+
+import (
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/operatorkit/controller"
+	"github.com/giantswarm/operatorkit/example/memcached-operator/logger"
+)
+
+// newSimpleResourceRouter creates a resource router which handles all objects
+// with single set of resources.
+func newSimpleResourceRouter(resources []controller.Resource) (*controller.ResourceRouter, error) {
```

Comment:
Extracted a function creating resource router always routing to single set of resources.

Reply:
Nice, good to separate this.